src/Models/TasksModel.php

```diff
@@ -1,21 +1,63 @@
 <?php
 
-require_once 'src/Services/DatabaseService.php';
+require_once 'src/Entities/TaskEntity.php';
+require_once 'src/Entities/UserEntity.php';
 
 class TasksModel
 {
-
     private PDO $db;
-
     public function __construct(PDO $db)
     {
         $this->db = $db;
     }
+    public function selectTaskById(int $taskId): TaskEntity
```

Comment:
If something happened that meant you asked for a taskID that doesn't exist, this method will return false.

fetch() will either return what you asked for, or false it it wasn't found.

You can make the return type hint `: TaskEntity|false` to achieve that